src/models/converters/transactions/summary.rs

```diff
@@ -35,19 +38,24 @@ impl MultisigTransaction {
         info_provider: &mut dyn InfoProvider,
     ) -> Result<Vec<TransactionSummary>> {
         let safe_info = info_provider.safe_info(&self.safe.to_string())?;
+        let tx_status = self.map_status(&safe_info);
+        let missing_signers = if tx_status == TransactionStatus::AwaitingConfirmations {
+            Some(self.missing_signers(&safe_info.owners))
+        } else {
+            None
+        };
```

Comment:
The way I was thinking to implement was to add an additional status. Otherwise we get this field in most cases as null, right? This could just be an additional `AwwaitingForOwnerConfirmation{ owner: String }` variant of the status enum. The clients would have to add additional deserialization logic, but at least we win in nullability, IMO

Reply:
See https://github.com/gnosis/safe-client-gateway/issues/150

Also `null` values are not rendered, so for most of the items nothing changes in the json. And this is way better to cache (else the cache depends on the owner). The client side overhead is minimal. This was communicated in slack and then the issue was updated.

Reply:
Yeah my bad, I didn't realise the issue was edited. I should have paid more attention.